src/pages/articles/canvas/particle-sim/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Particle Sim
+---
+## Particle Simulation in Canvas
+
+In this guide, we're going to build a basic particle simulation in Canvas using simple principles of animation.
+
+We will want to set up an array of particles with accelerations and velocities. We will create 100 particles at random points on the canvas.
+
+```
+var particles = [];
+for(var i=0; i<100; i++) {
+  particles.push(
+    {
+      x:Math.random()*canvas.width,
+      y:Math.random()*canvas.height,
+      vx:0,
+      vy:0,
+      ax:0,
+      ay:0
+    }
+  );
+}
+```
+
+In our draw loop, we render these particles.
+
+```
+function draw() {
+  ctx.clearRect(0, 0, canvas.width, canvas.height);
```

Comment:
Can you define `ctx` earlier in this article, some people might skip right ahead to this article. We don't need to explain why it is there (other articles can do that), but it will be good for completeness